instrumentation/jetty-httpclient/jetty-httpclient-9.2/library/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_2/JettyClientTracing.java

```diff
@@ -0,0 +1,76 @@
+package io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2;
+
+import static io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2.JettyClientWrapUtil.wrapResponseListeners;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import java.util.List;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.HttpClientTransport;
+import org.eclipse.jetty.client.HttpRequest;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Response;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+
+/**
+ * JettyClientTracing Designed for use outside of Javaagent. Use the static creation methods to
+ * build a Jetty HttpClient that is wrapped with tracing enabled
+ */
+public class JettyClientTracing extends HttpClient {
+
+  private final Instrumenter<Request, Response> instrumenter;
+
+  public static HttpClient create(OpenTelemetry openTelemetry) {
+    return new JettyClientTracing(openTelemetry);
+  }
+
+  public static HttpClient create(
```

Comment:
Let's add `JettyClientTracingBuilder` to avoid having too many factory methods

Reply:
Kinda lost me on this one.  Looking at the ArmeriaTracing, what I think I think you are saying is, create another builder with the role of building JettyClientTracing instances?